README.md

```diff
@@ -20,9 +20,10 @@
 
 OnnxSlim can help you slim your onnx model, with less operators, but same accuracy, better inference speed.
 
+- 🚀 2025/05/17: OnnxSlim is merged into [huggingface optimum](https://github.com/huggingface/optimum) 🤗🤗🤗
```

Comment:
```suggestion
- 🚀 2025/05/17: OnnxSlim is merged into [huggingface optimum](https://github.com/huggingface/optimum) 🤗🤗🤗
```
- 🚀 2025/05/17: OnnxSlim is merged into [optimum](https://github.com/huggingface/optimum) 🤗🤗🤗